### PR TITLE
Adds linux_aarch64 and linux_ppc64 builds

### PR DESCRIPTION
**Pull request by sanurielf**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by sanurielf** [comment]
@gerrymanoim @llllllllll @richafrank  when possible kindly review this so I can reopen the PR https://github.com/conda-forge/ta-lib-feedstock/pull/31 for a similar change in ta-lib.

**Comment by sanurielf** [comment]
@hmaarrfk can you take a look at this one? 🥇  Thanks!!

File: .ci_support/osx_arm64_.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ MACOSX_DEPLOYMENT_TARGET:
 c_compiler:
 - clang
 c_compiler_version:
-- '14'
+- '15'
```
Comment (hmaarrfk) [inline .ci_support/osx_arm64_.yaml:6]: you should bump the build number since OSX compiler version changed.

Comment (sanurielf) [reply .ci_support/osx_arm64_.yaml:6]: Damn, thanks! bumped up.

**Pull request merged**